experimental/cloudbuild/cloudbuild.yaml

```diff
@@ -128,4 +130,6 @@ substitutions:
   _FOURKEYS_PARSERS: '\"cloud-build\", \"github\", \"gitlab\", \"tekton\"'
   _FOURKEYS_PROVISION_BRANCH: 'main'
   _FOURKEYS_REGION: 'us-central1'
+  _FOURKEYS_DOMAIN_MAPPING_REGION: 'us-central1'
+  _FOURKEYS_MAPPED_DOMAIN: 'dora.nandos.dev'
```

Comment:
Not sure there should be a default and this needs to go into infrastructure repo.